cirq-core/cirq/transformers/routing/route_circuit_cqc_test.py

```diff
@@ -107,6 +107,45 @@ def test_circuit_with_measurement_gates():
     cirq.testing.assert_same_circuits(routed_circuit, circuit)
 
 
+def test_circuit_with_valid_intermediate_multi_qubit_measurement_gates():
```

Comment:
this test doesn't make sense since the measurements are terminal. Also this test should test that an intermediate 2-qubit test is valid. so try 
```py3
qs = cirq.LineQubit.range(2)
cirq.Circuit(cirq.measure_each(*q), cirq.H.on_each(q))
```